doc/api/synopsis.md

```diff
@@ -9,9 +9,43 @@ Please see the [Command Line Options][] document for information about
 different options and ways to run scripts with Node.js.
 
 ## Example
-
 An example of a [web server][] written with Node.js which responds with
-`'Hello World'`:
+`'Hello World!'`:
+
+Firstly, Make sure you have downloaded Node.js from [Node.js Official website](http://nodejs.org/#download).
+Then, Follow this [installation guide](https://nodejs.org/en/download/package-manager/).
+
+
+Now, Create an empty project folder called `projects`, navigate into it:
```

Comment:
Can you tweak this to make it clear that `projects` is just an example name, not one that has any significance (i.e. you don't _have_ to call it `projects`)

Maybe:

>If you aren't sure where to put your code, a good place to start is to create a `projects` folder in your home directory like so:

(
I took inspiration from the rust book again, specifically here:

>First, make a directory to put your Rust code in. Rust doesn’t care where your code lives, but for this book, we’d suggest making a projects directory in your home directory and keeping all your projects there. Open a terminal and enter the following commands to make a directory for this particular project:

https://doc.rust-lang.org/book/second-edition/ch01-02-hello-world.html#creating-a-project-directory
)